pkg/instance/handler.go

```diff
@@ -192,29 +192,31 @@ func (h Handler) Deploy(c *gin.Context) {
 	c.JSON(http.StatusCreated, instance)
 }
 
-func convertRequiredParameters(instanceID uint, requestParameters []ParameterRequest) []model.InstanceRequiredParameter {
+func convertRequiredParameters(instance *model.Instance, requestParameters []ParameterRequest) []model.InstanceRequiredParameter {
```

Comment:
Why did you change the parameter from uint to *model.Instance? All you need in here is the uint, right?